timesketch/lib/datastores/opensearch.py

```diff
@@ -114,7 +121,7 @@ class OpenSearchDataStore:
         "yellow"  # Minimum health status required ('yellow' or 'green')
     )
 
-    def __init__(self, host="127.0.0.1", port=9200):
+    def __init__(self, host="127.0.0.1", port=9200, **kwargs):
```

Comment:
Since you introduce `kwargs` as parameter, could you explain in the docstring, what is expected and what the keyword arguments are expected / might be provided?